homeassistant/components/ipma/config_flow.py

```diff
@@ -0,0 +1,52 @@
+"""Config flow to configure IPMA component."""
+import voluptuous as vol
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN, HOME_LOCATION_NAME
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class IpmaFlowHandler(data_entry_flow.FlowHandler):
+    """Config flow for SMHI component."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
+        """Init IpmaFlowHandler."""
+        super().__init__()
```

Comment:
There's no init method in the parent class.

Reply:
I was just following SMHI component as an example. Guessed it was forward thinking...

(removing)